yt_dlp/extractor/picarto.py

```diff
@@ -10,8 +10,8 @@
 
 
 class PicartoIE(InfoExtractor):
-    IE_NAME = 'picarto:base'
-    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[^/#?]+)$'
+    IE_NAME = 'picarto'
+    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[a-zA-Z0-9]+)/?(?:$|[?#])'
```

Comment:
you didn't have to revert to `[a-zA-Z0-9]+`, I was just curious
```suggestion
    _VALID_URL = r'https?://(?:www.)?picarto\.tv/(?P<id>[^/#?]+)/?(?:$|[?#])'
```

Reply:
My bad! Thanks for clarifying.